saleor/graphql/schema.graphql

```diff
@@ -1936,6 +1936,67 @@ type DraftOrderUpdate {
   order: Order
 }
 
+type EventDelivery implements Node {
+  id: ID!
+  createdAt: DateTime!
+  status: EventDeliveryStatusEnum!
+  eventType: WebhookEventTypeEnum!
+  attempts(sortBy: EventDeliverySortingInput, before: String, after: String, first: Int, last: Int): EventDeliveryAttemptCountableConnection
```

Comment:
We should add a dedicated type for sorting attempts, even if the sorting fields are the same for both attempts and deliveries for now. If we ever extend the `EventDelivery` type with some specific sorting fields that are not available in the `EventDeliveryAttempt` type, then `EventDeliverySortingInput` would be not suitable here.